src/layouts/hed__Application__c-K12 Kit Application Layout.layout

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Layout xmlns="http://soap.sforce.com/2006/04/metadata">
+    <layoutSections>
```

Comment:
@jofsky  @jstvz @lifewithryan , do I need to add a flexipage in the src folder for the new page layout that I've added in the src folder?

Reply:
@kdy618 How did you create the page layout? If you created it in an org and pulled it down, a flexipage should have pulled down with it (assuming we created the lightning page for it in app builder, etc...)

Reply:
@lifewithryan , nope I created the layout.layout file via vs code and push it to the scratch org.